bin/heudiconv

```diff
@@ -367,34 +367,42 @@ def conversion_info(subject, outdir, info, filegroup, basedir=None, ses=None):
     return convert_info
 
 
-def embed_nifti(dcmfiles, niftifile, infofile, bids_info=None, force=False):
-    import dcmstack as ds
+def embed_nifti(dcmfiles, niftifile, infofile, bids_info=None, force=False, min_meta=False):
     import nibabel as nb
     import os
     import json
-    stack = ds.parse_and_stack(dcmfiles, force=force).values()
-    if len(stack) > 1:
-        raise ValueError('Found multiple series')
-    stack = stack[0]
-    #Create the nifti image using the data array
-    if not os.path.exists(niftifile):
-        nifti_image = stack.to_nifti(embed_meta=True)
-        nifti_image.to_filename(niftifile)
-        return ds.NiftiWrapper(nifti_image).meta_ext.to_json()
-    orig_nii = nb.load(niftifile)
-    aff = orig_nii.get_affine()
-    ornt = nb.orientations.io_orientation(aff)
-    axcodes = nb.orientations.ornt2axcodes(ornt)
-    new_nii = stack.to_nifti(voxel_order=''.join(axcodes), embed_meta=True)
-    meta = ds.NiftiWrapper(new_nii).meta_ext.to_json()
-    meta_info = json.loads(meta)
+    import dcmstack as ds
```

Comment:
I was thinking this to potentially bypass a dependency, but it's also imported in `process_dicoms`. I still agree just so we do one less import in the node